fs/xfstests.py

```diff
@@ -221,6 +229,72 @@ def setUp(self):
         self.test_mnt = self.params.get('test_mnt', default='/mnt/test')
         self.disk_mnt = self.params.get('disk_mnt', default='/mnt/loop_device')
         self.fs_to_test = self.params.get('fs', default='ext4')
+        self.run_type = self.params.get('run_type', default='distro')
+
+        if self.run_type == 'upstream':
+            if self.fs_to_test == "ext4":
+                # Build e2fs progs
+                e2fsprogs_dir = os.path.join(self.teststmpdir, 'e2fsprogs')
+                if not os.path.exists(e2fsprogs_dir):
+                    os.makedirs(e2fsprogs_dir)
+                e2fsprogs_url = self.params.get('e2fsprogs_url')
+                git.get_repo(e2fsprogs_url, destination_dir=e2fsprogs_dir)
+                e2fsprogs_build_dir = os.path.join(e2fsprogs_dir, 'build')
+                if not os.path.exists(e2fsprogs_build_dir):
+                    os.makedirs(e2fsprogs_build_dir)
+                os.chdir(e2fsprogs_build_dir)
+                process.run("../configure", verbose=True)
+                build.make(e2fsprogs_build_dir)
+                build.make(e2fsprogs_build_dir, extra_args='install')
```

Comment:
Would this command install the binaries in /usr/ or /usr/local ?

Reply:
I noticed that this installs in `/sbin` (although docs suggested `/usr/local/bin` however I do agree that it would be better to explitly pass `--prefix=/usr/local` as `/usr/local/bin` seems to precede `/sbin` in `$PATH`. 

I'll do the same for xfsprogs as well.